geomstats/geometry/hyperbolic.py

```diff
@@ -632,7 +634,7 @@ def log(self, point, base_point):
             point = gs.to_ndarray(point, to_ndim=2)
             base_point = gs.to_ndarray(base_point, to_ndim=2)
 
-            angle = self.dist(base_point, point)
+            angle = self.dist(base_point, point) / self.scale
```

Comment:
Ah! That was the self.scale factor that we were missing?

Reply:
Yes !